test/tools/CodeCoverageAutomation/Start-CodeCoverageRun.ps1

```diff
@@ -145,8 +145,16 @@ $jsonFile = "$outputBaseFolder\CC.json"
 
 try
 {
+    ## Github needs TLS1.2 whereas the defaults for Invoke-WebRequest do not have TLS1.2
+    $prevSecProtocol = [System.Net.ServicePointManager]::SecurityProtocol
+
+    [System.Net.ServicePointManager]::SecurityProtocol =
+        [System.Net.ServicePointManager]::SecurityProtocol -bor
```

Comment:
don't or it with the existing setting.  The SDL say 1.0 and newer at most.

Reply:
The default is TLS 1.0.